api/src/main/java/uk/gov/dhsc/htbhf/claimant/scheduler/MessageProcessorScheduler.java

```diff
@@ -88,7 +88,7 @@ public void processPaymentMessages() {
         messageProcessor.processMessagesOfType(MessageType.MAKE_PAYMENT);
     }
 
-    @Scheduled(cron = EVERY_30_SECONDS)
+    @Scheduled(cron = "05/35 * * * * *") // every 30 seconds, offset from other schedules by 5 seconds to prevent clashes with CREATE_NEW_CARD for new claims
```

Comment:
Again, unrelated, but I saw that for new claims the CREATE_NEW_CARD and REPORT_CLAIM messages were always clashing and causing one of them to fail with a hibernate concurrent update exception. This offsets reporting new claims by 5 seconds to prevent that.